packages/config/src/config/routes.ts

```diff
@@ -73,15 +55,13 @@ export default function getRoutes(): ServerRoute[] {
       options: {
         auth: {
           scope: [
-            RouteScope.NATLSYSADMIN,
-            RouteScope.DECLARE,
-            RouteScope.REGISTER,
-            RouteScope.CERTIFY,
-            RouteScope.PERFORMANCE,
-            RouteScope.SYSADMIN,
-            RouteScope.VALIDATE,
-            // @TODO: Refer to an enum / constant
-            'record.confirm-registration'
+            SCOPES.CONFIG_UPDATE_ALL,
+            SCOPES.RECORD_DECLARE_BIRTH,
+            SCOPES.RECORD_DECLARE_DEATH,
+            SCOPES.RECORD_DECLARE_MARRIAGE,
+            SCOPES.RECORD_REGISTER,
+            SCOPES.PERFORMANCE_READ,
+            SCOPES.RECORD_SUBMIT_FOR_APPROVAL
```

Comment:
This route is probably required by all our users right? So maybe not restricting it to any scopes is a better choice?